---
layout: issue
title: "Call to undefined function Zend_Exception()  in Zend/Test/PHPUnit/Db/Operation/Truncate.php on line 107"
id: ZF-12129
---

ZF-12129: Call to undefined function Zend\_Exception() in Zend/Test/PHPUnit/Db/Operation/Truncate.php on line 107
-----------------------------------------------------------------------------------------------------------------

 Issue Type: Bug Created: 2012-03-31T02:39:05.000+0000 Last Updated: 2012-03-31T02:48:45.000+0000 Status: Open Fix version(s): 
 Reporter:  Justin Fluhmann (jfluhmann)  Assignee:  Matthew Weier O'Phinney (matthew)  Tags: - Zend\_Test\_PHPUnit
 
 Related issues: 
 Attachments: 
### Description

line 107 reads: throw Zend\_Exception("IBM Db2 TRUNCATE not supported.");

should read: throw new Zend\_Exception("IBM Db2 TRUNCATE not supported.");

 

 

### Comments

No comments to display